src/wp-includes/class-wp-theme-json.php

```diff
@@ -2713,56 +2714,81 @@ private static function get_block_nodes( $theme_json, $selectors = array(), $opt
 		}
 
 		foreach ( $theme_json['styles']['blocks'] as $name => $node ) {
-			$selector = null;
-			if ( isset( $selectors[ $name ]['selector'] ) ) {
-				$selector = $selectors[ $name ]['selector'];
-			}
 
-			$duotone_selector = null;
-			if ( isset( $selectors[ $name ]['duotone'] ) ) {
-				$duotone_selector = $selectors[ $name ]['duotone'];
-			}
+			$include_node_paths_only = $options['include_node_paths_only'] ?? false;
+			$node_path = array( 'styles', 'blocks', $name );
+			if ( $include_node_paths_only ) {
```

Comment:
Will there be unit tests for the changes in this method?

Reply:
Back ported in https://github.com/WordPress/wordpress-develop/pull/7486/commits/35e4ed432212d8c2fb984efe86e968662dc83ba9